src/commissaire_http/handlers/hosts.py

```diff
@@ -187,6 +189,12 @@ def create_host(message, bus):
         # pass this off to the investigator
         bus.notify('jobs.investigate', params={'address': address})
 
+        # Push the host to the Watcher queue
+        watcher_record = models.WatcherRecord(
+            address=address,
+            last_check=_dt.utcnow().isoformat())
```

Comment:
Of course I think of this after having just merged the model, but a `WatcherRecord.new()` method that fills in `last_check` as you're doing here would be handy.  Not super important, and certainly not a blocker.